lib/crypto/sr25519/sr25519.go

```diff
@@ -154,6 +159,30 @@ func NewPrivateKey(in []byte) (*PrivateKey, error) {
 	return priv, err
 }
 
+// NewPrivateKeyFromHex returns a private key from a seed
+func NewPrivateKeyFromHex(seed string) (*PrivateKey, error) {
```

Comment:
this doesn't look like it uses the input as a seed but directly turns it into a private key? maybe rename `seed` to `keyStr` or something

Reply:
sounds good!